src/buildercore/bakery.py

```diff
@@ -13,6 +13,8 @@ def ami_name(stackname):
 @core.requires_active_stack
 def create_ami(stackname, name=None):
     "creates an AMI from the running stack"
+    # NOTE: alfred is able to run this task as BOOTSTRAP_USER because it created the instance (somehow).
+    # or I downloaded the key for it (I don't recall).
```

Comment:
improve comment. does elife-alfred user have access to the basebox instance or not?